src/components/ConnectorList/ConnectorAccordion/index.tsx

```diff
@@ -15,36 +15,40 @@ import ReleaseAccordion, { Release } from '@site/src/components/ConnectorList/Re
 import { Paper } from '@mui/material';
 
 export class Connector {
-  nameWithOwner: string;
+  name_with_owner: string;
```

Comment:
Why changing these to  snake case?. Camel case is more common in JS / React. Personally, I'd like to see these as camel case.

Reply:
I personally don't care so much one way or the other, I changed it to make it consistent, because we mixed them both (see nested structs). If you feel strongly I can go in and change the nested structs instead 🤷

Reply:
One thing I'll add is that GitHub uses snake case in their API ([example](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#get-a-repository)), and since this data comes directly from their API, keeping it in snake case would better align with their data model. I'm trying to avoid extra work here 🙈